rasa_sdk/tracing/utils.py

```diff
@@ -17,11 +18,12 @@ def get_tracer_provider(endpoints_file: str) -> Optional[TracerProvider]:
 
 
 def get_tracer_and_context(
-    tracer_provider: Optional[TracerProvider], request: Request
+    tracer_provider: Optional[TracerProvider], request: Union[Request, WebhookRequest]
 ) -> Tuple[Any, Any, Text]:
     """Gets tracer and context."""
     span_name = "create_app.webhook"
-    if tracer_provider is None:
+
+    if tracer_provider is None or isinstance(request, WebhookRequest):
```

Comment:
Yeah we will need to update the code on rasa-private side too to inject the trace context in the grpc request. I am searching for some docs / looking in opentelemetry to see if there's any guidance.

Reply:
Couldn't find any guidance I'm afraid :(

Reply:
I'll try to look it up.

Reply:
@ancalita I added tracing for `GRPCActionServerWebhook.Webhook`.